packages/@aws-cdk/aws-synthetics/test/integ.canary.expected.json

```diff
@@ -119,7 +119,10 @@
           "DurationInSeconds": "0",
           "Expression": "rate(1 minute)"
         },
-        "StartCanaryAfterCreation": true
+        "StartCanaryAfterCreation": true,
+        "RunConfig": {
+          "TimeoutInSeconds": 60
```

Comment:
This is a relic of the previous PR, is this behavior that we want? Or should Timeout only be introduced when any other `RunConfig` properties are specified?